src/jenkins_job_insight/analyzer.py

```
@@ -40,6 +40,45 @@
+    """
+    if not repo_path:
+        return ""
+
+    prompt_path = repo_path / JOB_INSIGHT_PROMPT_FILENAME
+    if not prompt_path.is_file():
+        return ""
+
+    try:
+        content = prompt_path.read_text(encoding="utf-8").strip()
+        logger.info("Using custom prompt from repo: %s", prompt_path)
+        return content
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)